definitions/npm/tempy_v3.x.x/test_tempy.js

```diff
@@ -0,0 +1,58 @@
+// @flow
```

Comment:
Regarding test would it be possible to add in the `describe/it` format? Would make it easier to differentiate use cases in the future for extending

Reply:
Thanks for the feedback. I've updated the test file with `describe`/`it`.